src/locales/en.js

```diff
@@ -318,7 +318,7 @@ export default {
     'menu.details': 'Details',
     'menu.groups': 'Groups',
     'menu.database': 'Database',
-    'menu.widget': 'MFE & Widgets',
+    'menu.widget': 'Widgets',
```

Comment:
this value should remain 'MFE & Widgets'

Reply:
The designs shows only Widgets, but I'll roll it back